models/lioncore.json

```diff
@@ -0,0 +1,661 @@
+[
+  {
+    "type": "jj0Y6KYlHt3_qyFDFAx4CA",
+    "id": "31_xAp4rmNVj6Sdahgjeuw",
+    "properties": {
+      "97hsg4Fg3ZitV5FaapRLVg": "LIonCore"
```

Comment:
Hard-code human readable IDs (I think @enikao suggested them in another discussion) could make the file more readable but also help get compatibility with other implementation (e.g., the Java one) easier

Reply:
That's a good idea. I made issue #19 for this, as I think this requires a bit of discussion.